pkg/datapath/loader/base.go

```diff
@@ -351,6 +351,15 @@ func (l *Loader) Reinitialize(ctx context.Context, o datapath.BaseProgramOwner,
 	}).Info("Setting up BPF datapath")
 
 	if option.Config.IPAM == ipamOption.IPAMENI {
+		// For the ENI ipam mode on EKS, this will be the interface that
+		// the router (cilium_host) IP is associated to.
+		if info := node.GetRouterEniInfo(); info != nil {
+			iface, err := linuxrouting.RetrieveIfaceNameFromMAC(info.MasterIfMAC.String())
+			if err != nil {
+				log.WithError(err).Fatalf("failed to set encrypt interface in the ENI ipam mode")
```

Comment:
This should use `Fatal()`, not `Fatalf()`. Also, I think we're supposed to capitalize the first word in log messages (Note, this is for log messages, not error messages; error messages should be all lower case).

Reply:
(I also wondered about whether `Fatal` is appropriate here, but it looks like `Fatal` is already used at this point in the daemon so introducing one more usage in the case where we cannot fetch the MAC seems unlikely to incur additional friction for users)